README.md

```diff
@@ -19,6 +19,9 @@ Based on a [paketo rails-assets](https://github.com/paketo-buildpacks/rails-asse
 "bundle exec rake assets:precompile assets:clean"
 ```
 
+Buildpack tries to reuse cached layer with assets if SHA256 checksum of assets directories shown above wasn't changed since last run. This behaviour may be changed by setting environment variable `RAILS_ASSETS_DISABLE_CACHING=TRUE`.
```

Comment:
The env variable `RAILS_ASSETS_DISABLE_CACHING` should be set to `true` by default. I don't see it being set to true in your code. That means, Caching is enabled by default which is not what we want.

Reply:
Made all changes and also fixed tests.